SB2018/2018-01-12-2.md

```diff
@@ -0,0 +1,34 @@
+---
+to: Southend
+subject: SourceBots Mentoring and Tech Days
+---
+
+Hi all,
+
+We hope you're progressing well on your robot!
+We have some important things to announce, so keep reading:
+
+## Mentoring
+
+We're not sure if we can provide mentoring for you, as you are quite far away from most of our possible mentors, however if you really need someone to mentor you, we'll see what we can do.
```

Comment:
I'd make sure to reinforce here (and in the main email) that mentoring doesnt have to be in-person. Not only is there hangouts etc, but support can be provided through the `techsupport@` address